src/main.rs

```diff
@@ -76,12 +76,14 @@ fn main() {
         .add_project("https://github.com/coredump-ch/")
         .add_cam("https://webcam.coredump.ch/cams/ultimaker_0.jpg")
         .add_extension("ccc", "chaostreff")
+        .state(api::State {
+            open: Some(false),
```

Comment:
Shouldn't this be `None` by default?

Reply:
Debatable. But if you look at the message it is the closed message.

Reply:
But you are right: The `StateFromPeopleNowPresent` will set it to `true` / `false` depending on the sensor value or leave it `None` if no sensor is available, which is nice.

Reply:
Fixed.